src/server/index.ts

```diff
@@ -142,17 +142,32 @@ export const setupServer = () => {
     );
   });
 
+  // Add Swagger UI route
+  app.get("/doc", swaggerUI({ url: "/openapi.json" }));
+
+  // Serve OpenAPI JSON
+  app.get("/openapi.json", (c) => {
+    return c.json(app.getOpenAPIDocument({
+      openapi: "3.0.0",
+      info: {
+        version: "1.0.0",
+        title: "HDR Browser API",
+      },
+    }));
+  });
+
+  // Add OpenAPI document metadata
   app.doc("/doc", {
     openapi: "3.0.0",
     info: {
       version: "1.0.0",
       title: "HDR Browser API",
     },
   });
+
   app.route("/", pageRouter);
   app.route("/", browserRouter);
 
-  app.get("/", swaggerUI({ url: "/doc" }));
```

Comment:
We reference this in src/bin/serve.ts as a string for people to reference. If we're going to remove it, can we add the new location there?